src/test/java/christmas/domain/DateTest.java

```diff
@@ -0,0 +1,36 @@
+package christmas.domain;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import christmas.utils.validators.DateValidator;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+
+class DateTest {
+    private final DateValidator dateValidator = new DateValidator();
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3, 4})
+    void 정수_날짜를주면_Date객체를_생성한다(int day) {
+        Date result = Date.from(day, dateValidator);
+        assertThat(result).isInstanceOf(Date.class);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {32, 0, -1, 35})
+    void 달력숫자외의_숫자를주면_예외를_반환한다(int day) {
+        assertThatThrownBy(() -> Date.from(day, dateValidator)).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest
```

Comment:
아주 사소하지만, `@ParameterizedTest`에도 이름을 지정해 줄 수 있으며 이 경우 테스트 케이스 실패에 대해 어떤 테스트 케이스가 실패했는지 조금 더 빠르게 파악할 수 있다는 장점이 있습니다!
```suggestion
    @ParameterizedTest(name = "{0}일의 주말 판단 결과는 {1} 이다.")
```